### PR TITLE
Add a Gemfile

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by lzap** [comment]
Will the RPM build? +1

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem 'mime-types', '<2.0'  # Needs ruby 1.9.2+
```
Comment (ekohl) [inline Gemfile:3]: This is an indirect dependency btw. Not sure if you can check for the ruby version in a Gemfile and only then set it.

**Comment by ekohl** [comment]
I did find out that running the installer under bundler can give bad results. Things like puppet fail to run properly so I'm not sure if this is still needed.

**Comment by domcleal** [comment]
The RPM builds: http://ci.theforeman.org/job/packaging_build_matrix_rpm/854/

File: Gemfile (hunk 1 above)
Comment (domcleal) [inline Gemfile:3]: `gem 'mime-types', :platforms => :ruby_18`

**Comment by ekohl** [comment]
I don't think this is needed anymore and needs a rebase anyway.

**Pull request closed without merging**